src/react.fs

```diff
@@ -19,24 +19,45 @@ module Helpers =
 module Program =
     open Fable.Import.Browser
 
+    // TODO: Provide an alternative for browsers that don't support it
+    let private now() = performance.now()
+
+    type private Dispatcher<'Value>(f: 'Value -> unit) =
```

Comment:
`'Value` probably should be `'model` - I understand the Dispatcher doesn't care, but since it's implemented here the name might as well be revealing.

Reply:
Changing it should be no problem (probably it would be also a good idea to give more semantics to the type name in case we want to use it somewhere else, RafDispatcher?), although I'm actually passing a tuple containing both model and dispatch as `'Value` right now.